test/uk/gov/hmrc/tai/connectors/EmploymentsConnectorSpec.scala

```diff
@@ -246,39 +221,36 @@ class EmploymentsConnectorSpec
 
     "return none" when {
       "invalid json returned by an api" in {
-        val sut = createSUT()
         when(httpHandler.getFromApiV2(any())(any())).thenReturn(Future.successful(Json.parse(zeroEmployments)))
 
-        Await.result(sut.employment(nino, "123"), 5.seconds) mustBe None
+        Await.result(sut().employment(nino, "123"), 5.seconds) mustBe None
       }
     }
   }
 
   "EmploymentsConnector endEmployment" must {
     "return an envelope" when {
       "we send a PUT request to backend" in {
-        val sut = createSUT()
         val json = Json.obj("data" -> JsString("123-456-789"))
         when(httpHandler.putToApi(any(), any())(any(), any(), any()))
-          .thenReturn(Future.successful(HttpResponse(200, Some(json))))
+          .thenReturn(Future.successful(HttpResponse(200, json, Map[String, Seq[String]]())))
 
         val endEmploymentData = EndEmployment(new LocalDate(2017, 10, 15), "YES", Some("EXT-TEST"))
 
-        val result = Await.result(sut.endEmployment(nino, 1, endEmploymentData), 5.seconds)
+        val result = Await.result(sut().endEmployment(nino, 1, endEmploymentData), 5.seconds)
 
         result mustBe "123-456-789"
       }
     }
 
     "return an exception" when {
       "json is invalid" in {
-        val sut = createSUT()
         val json = Json.obj("test" -> JsString("123-456-789"))
         when(httpHandler.putToApi(any(), any())(any(), any(), any()))
-          .thenReturn(Future.successful(HttpResponse(200, Some(json))))
+          .thenReturn(Future.successful(HttpResponse(200, json, Map[String, Seq[String]]())))
```

Comment:
`Map[String, Seq[String]]()` could just be `Map.empty()` here?

Reply:
If I don't declare the types in the Map I get a compilation error